helpers/helpers.v1.d/go

```diff
@@ -226,29 +235,37 @@ ynh_remove_go() {
 ynh_cleanup_go() {
 
     # List required Go versions
-    local installed_apps=$(yunohost app list --output-as json --quiet | jq -r .apps[].id)
+    local installed_apps=$(yunohost app list | grep -oP 'id: \K.*$')
```

Comment:
Uuuuh not sure why switching to the new version ... the current one with json and `jq` seems much more clean and robust ? Is it that the ruby helpers still use an hacky grep ?

Reply:
Yes ruby helper still using grep
https://github.com/YunoHost/yunohost/blob/f7ed895c1811d13e444e043b0c9ace6401518bbc/helpers/helpers.v2.1.d/ruby#L231
and
https://github.com/YunoHost/yunohost/blob/f7ed895c1811d13e444e043b0c9ace6401518bbc/helpers/helpers.v1.d/ruby#L285